src/routes/classes/+page.svelte

```
@@ -153,51 +313,156 @@
 							<ArrowLeft size={16} />
 						</button>
-						<h3 class="text-lg font-semibold text-base-content">{selectedClass.name}</h3>
-						<div class="text-xs text-base-content/60 font-mono">{selectedClass.code}</div>
+						<div class="flex items-center gap-2 min-w-0">
+							<h3 class="text-lg font-semibold text-base-content truncate">{selectedClass.name}</h3>
+							<span class="badge badge-soft badge-sm rounded-full font-mono opacity-60 shrink-0">{selectedClass.code}</span>
+						</div>
 					</div>
 
+					<a
+						href={`/classes/${selectedClass._id}/tests/new`}
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)